samples/demo/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "demo",
+  "version": "0.0.1",
+  "author": "Ionic Framework",
+  "homepage": "https://ionicframework.com/",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/common": "~13.0.0",
+    "@angular/core": "~13.0.0",
+    "@angular/forms": "~13.0.0",
+    "@angular/platform-browser": "~13.0.0",
+    "@angular/platform-browser-dynamic": "~13.0.0",
+    "@angular/router": "~13.0.0",
+    "@capacitor/android": "3.3.4",
+    "@capacitor/app": "1.0.7",
+    "@capacitor/camera": "^1.2.3",
+    "@capacitor/core": "3.3.4",
+    "@capacitor/filesystem": "^1.0.7",
+    "@capacitor/haptics": "1.1.3",
+    "@capacitor/ios": "3.3.4",
+    "@capacitor/keyboard": "1.2.0",
+    "@capacitor/status-bar": "1.0.6",
+    "@capacitor/storage": "^1.2.3",
+    "@ionic/angular": "^6.0.0",
+    "@ionic/pwa-elements": "^3.1.1",
+    "@types/fs-extra": "^9.0.13",
+    "browserfs": "^1.4.3",
+    "crypto": "^1.0.1",
+    "crypto-browserify": "^3.12.0",
+    "fs": "^0.0.1-security",
+    "moment": "^2.29.1",
+    "node-polyfill-webpack-plugin": "^1.1.4",
+    "path": "^0.12.7",
+    "path-browserify": "^1.0.1",
+    "rxjs": "~6.6.0",
+    "stream-browserify": "^3.0.0",
+    "stream-http": "^3.2.0",
+    "tslib": "^2.2.0",
+    "zone.js": "~0.11.4"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~13.0.1",
+    "@angular-eslint/builder": "~13.0.1",
+    "@angular-eslint/eslint-plugin": "~13.0.1",
+    "@angular-eslint/eslint-plugin-template": "~13.0.1",
+    "@angular-eslint/template-parser": "~13.0.1",
+    "@angular/cli": "~13.0.1",
+    "@angular/compiler": "~13.0.0",
+    "@angular/compiler-cli": "~13.0.0",
+    "@angular/language-service": "~13.0.0",
+    "@capacitor/cli": "3.3.4",
+    "@dchagastelles/elastos-hive-js-sdk": "^2.4.4",
```

Comment:
The official package name is `@elastosfoundation/elastos-hive-js-sdk`. `@dchagastelles/elastos-hive-js-sdk` is used only for validation releases.

Reply:
Will update this. The official name is @elastosfoundation/hive-js-sdk.